libs/documentation/src/lib/components/result-list/demos/basic/result-list-basic.component.ts

```diff
@@ -1,21 +1,61 @@
-import { Component, OnInit, ViewChild, AfterViewInit, ChangeDetectorRef } from '@angular/core';
+import { Component, ChangeDetectorRef } from '@angular/core';
 
 @Component({
-  templateUrl: './result-list-basic.component.html',
+  templateUrl: './result-list-basic.component.html'
```

Comment:
clean up by making dynamic following similar pattern to https://angular.io/tutorial/toh-pt2